### PR TITLE
fix: rename label (close #474)

### LINKED ISSUE #474: Change `else` label to `other` on the search page
- Environment: https://mimirapp.xyz/search
- Client: desktop/reader

### ✔️ Expected behavior

How it should work

Should be `other` category label instead of `else`

### ✖️ Actual behavior

How it works now

Screenshot 2022-11-03 at 14 25 56

### Screenshot / Descriptions
https://www.figma.com/file/pxj9Hhk5AdOxXWJP0dq4gT/Library-App-Spec?node-id=105%3A442

### Note:
Add internalization for new text with the `localization` package (check if such translation already exists in the package)

### PR DESCRIPTION
**Pull request by vyacheslav1618**
#closes 474
Снимок экрана 2022-11-05 190705
Снимок экрана 2022-11-08 143559

### CODE REVIEW

**Comment by vyacheslav1618** [comment]
Снимок экрана 2022-11-05 190705

**Comment by Chichmarin17** [comment]
Looks ok for me. But could you add screenshots as well as the `closes` label right into pr's description rather that comments?

File: packages/frontapp/src/app/components/CategoriesList/index.tsx [hunk 1]
```diff
@@ -63,7 +64,7 @@ const CategoriesList: FC<ICategoriesListProps> = ({ allCategories }) => {
   return (
     <CategoriesWrapper>
       <Categories flex="2">
-        <Header>Books</Header>
+        <Header>{t('Books')}</Header>
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/CategoriesList/index.tsx:67]: This translation won't work when you switch to Russian (check how we use translation in other components)

File: packages/frontapp/src/app/components/CategoriesList/index.tsx [hunk 2]
```diff
@@ -78,7 +79,7 @@ const CategoriesList: FC<ICategoriesListProps> = ({ allCategories }) => {
         </List>
       </Categories>
       <Categories flex="1">
-        <Header>Else</Header>
+        <Header>{t('Other')}</Header>
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/CategoriesList/index.tsx:82]: Here as well

**Review by Chichmarin17 — CHANGES_REQUESTED** [review]

**Review by Chichmarin17 — APPROVED** [review]

**Pull request merged**